src/reducers/signupReducer.js

```diff
@@ -0,0 +1,17 @@
+import * as types from '../actions/actionTypes';
```

Comment:
The thing with this new reducer is that, and I know it's not well explained or documented anywhere in this repo, `redux-react-session` already stores an `authenticated` boolean. It also persists it in Local Storage so it remains among sessions.

Reply:
Makes sense! 👍